tests/logs/community_push/test_community_push/test_do_community_push_skip_houly_sap_ha[SAP].txt

```diff
@@ -2,6 +2,7 @@
 [    INFO] Retrieving the mappings for ami_pushitem from https://starmap-example.com using the community workflow.
 [    INFO] starmap query returned for ami_pushitem : {"name": "test-build", "version": "7.0", "query_response": {"name": "SAP", "workflow": "community", "clouds": {"aws_storage": [{"id": null, "meta": {"billing-code-config": {"sample-hourly": {"name": "Hourly2", "codes": ["bp-6fa54006"], "image_name": "sample_product", "image_types": ["hourly"]}, "sample-access": {"name": "Access2", "codes": ["bp-63a5400a"], "image_name": "sample_product", "image_types": ["access"]}}}, "architecture": "x86_64", "destination": "fake-destination-access", "overwrite": false, "restrict_version": false, "restrict_major": null, "restrict_minor": null, "provider": "awstest", "tags": null}]}}} 
 [ WARNING] No accounts definition in StArMap, leaving the defaults from credentials.
+[ WARNING] No sharing_accounts definition in StArMap, leaving the defaults from credentials.
```

Comment:
As a consequence of adding support for both `accounts` and `sharing_accounts` one of them may not be present which will cause this warning... Please let me know if you want me to suppress it somehow..

Also if the mapping provides both `accounts` and `sharing_accounts` the latter may get ignored due to the `setdefault` nature which will get just the values from `accounts`... If this is a problem I can try to find a way to fix as well